src/NzbDrone.Core/NetImport/RSSImport/RSSImportParser.cs

```diff
@@ -146,7 +146,7 @@ protected virtual Movie ProcessItem(XElement item, Movie releaseInfo)
 
             if (result != null)
             {
-                releaseInfo.Title = result.MovieTitle;
+                releaseInfo.Title = result.PrimaryMovieTitle;
```

Comment:
Wasn't sure what else would make sense here with multiple possible titles - should I fill in `AlternativeTitles`?